src/common/internal/logging/test_case_recorder.ts

```diff
@@ -56,20 +72,13 @@ export class TestCaseRecorder {
     }
 
     // Convert numeric enum back to string (but expose 'exception' as 'fail')
-    this.result.status =
-      this.finalCaseStatus === LogSeverity.Pass
-        ? 'pass'
-        : this.finalCaseStatus === LogSeverity.Skip
-        ? 'skip'
-        : this.finalCaseStatus === LogSeverity.Warn
-        ? 'warn'
-        : 'fail'; // Everything else is an error
+    this.result.status = logSeverityToString(this.finalCaseStatus);
```

Comment:
Could the final result status ever be "notrun" ?

It might be weird to ever see that reported. We could consider asserting it is never notrun 

In Chrome, telemetry will silently treat this as Pass. Chrome checks for "skip" and "fail". Other stuff is considered pass (can be changed of course).

Reply:
It is 'notrun' if you don't run anything and this is tested in `logger.spec.ts`

We should probably fix telemetry. Assuming pass doesn't seem like a good default.